sidebars.js

```diff
@@ -405,13 +402,10 @@ module.exports = {
   "Self-Managed": [
     "self-managed/overview",
     {
-      "Installation": [
+      Installation: [
         "self-managed/platform-deployment/index",
         {
-          "Kubernetes": [
-            "self-managed/platform-deployment/kubernetes",
-            "self-managed/platform-deployment/kubernetes-helm",
-          ],
+          Kubernetes: ["self-managed/platform-deployment/kubernetes"],
```

Comment:
Potentially blocking: 

There's one removed item here (`kubernetes-helm`) -- is that intentional?

Reply:
Nope, good catch. Might have happened in the conflict resolution